main.go

```diff
@@ -67,6 +67,8 @@ type CmdLineOpts struct {
 	help                   bool
 	version                bool
 	kubeSubnetMgr          bool
+	kubeApiUrl             string
```

Comment:
nit: it's a golang best practice to keep all the acronym characters same case in a variable name, but not a big deal